apps/web/app/api/deps/route.ts

```diff
@@ -0,0 +1,5 @@
+import { getAppDepsHandler } from '@verity/deps-api';
```

Comment:
Maybe we can use names like [in this repo](https://github.com/Saasfy/saasfy/tree/main/packages/crud), for example?